src/org/zalando/stups/friboo/auth.clj

```diff
@@ -0,0 +1,39 @@
+(ns org.zalando.stups.friboo.auth
+  (:require [clj-http.client :as http]
+            [org.zalando.stups.friboo.ring :as r]
+            [org.zalando.stups.friboo.config :refer [require-config]]
+            [org.zalando.stups.friboo.log :as log]
+            [io.sarnowski.swagger1st.util.api :as api]
+            [com.netflix.hystrix.core :as hystrix :refer [defcommand]]))
+
+(defcommand fetch-auth
+  "Asks magnificent if the user for this token has access to resources of this team. Returns true or false."
+  {:hystrix/fallback-fn (constantly false)}
```

Comment:
~~~
=> (defcommand foo {:hystrix/fallback-fn (constantly false)} (throw (new Exception "foo")))

=> (foo)
false
~~~

Hystrix itself doesn't log the fallback at all, apparently. I will add some.